docs/en/connector-v2/source/Elasticsearch.md

```diff
@@ -30,7 +30,7 @@ support version >= 2.x and <= 8.x.
 | index_list              | array   | no       | used to define a multiple table task                           |
 | source                  | array   | no       | -                                                              |
 | query                   | json    | no       | {"match_all": {}}                                              |
-| search_type             | json    | no       | Search method,sql or dsl,default dsl                           |
+| search_type             | json    | no       | Search method,SQL、PIT、SCROLL,default SCROLL                    |
```

Comment:
`search_type` and `search_api_type` not same.
`search_type` : SQL, DSL
`search_api_type`: SCROLL, PIT